deploy/helm/scf/assets/operations/instance_groups/scheduler.yaml

```diff
@@ -123,10 +102,13 @@
     - name: ssh-proxy
       protocol: TCP
       internal: 2222
-    run:
-      memory: 256
-      virtual-cpus: 2
 
 - type: replace
   path: /instance_groups/name=scheduler/jobs/name=cfdot/properties/quarks?/bpm/processes
   value: []
+
+- type: replace
+  path: /variables/name=diego_auctioneer_server/options/alternative_names?
+  value:
+  - "((deployment-name))-scheduler"
+  - "*.((deployment-name))-scheduler"
```

Comment:
Same as before, comment it out and leave a TODO. We shall eventually get it working if we make the custom CoreDNS to work in our favour.